src/FakeAuth/HttpClientExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using FakeAuth.Profiles;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Claims;
+
+namespace FakeAuth
+{
+	public static class HttpClientExtensions
+	{
+		public static void SetFakeAuthClaimns<TProfile>(this HttpClient client) where TProfile : IFakeAuthProfile, new()

```

Comment:
SetFakeAuthClaims - now with IFakeAuthProfile support!

Reply:
This makes me happy.  Tests that use SetFakeAuthClaims, can re-use any profiles that might be used in the startup of an app.  And since the handler will overwrite any config set claims with HttpClient claims it all just works. 🤩

Reply:
✨ new and improved! ✨